Winetricks does not open

Winetricks does not open when you select it from the drop-down menu. But if you run lutris from the terminal using the lutris command, Winetricks opens when you select it from the dropdown menu.

Arch Linux
![image](https://user-images.githubusercontent.com/53097257/164336430-93c7eadb-1bdc-44c2-8fec-7a8182a0a9f2.png)

> Was having the issue with it not working because -> ` zenity: symbol lookup error: /usr/lib/libwebkit2gtk-4.0.so.37: undefined symbol: gst_codec_utils_caps_get_mime_codec` So I thought I'd search for a zenity fork and I found there is [zenity-light](https://aur.archlinux.org/packages/zenity-light) that doesn't use webkit, works just fine! Perhaps this could be bundled with lutris own winetricks?

Thanks a lot, that fixed the issue for me!

> Was having the issue with it not working because -> ` zenity: symbol lookup error: /usr/lib/libwebkit2gtk-4.0.so.37: undefined symbol: gst_codec_utils_caps_get_mime_codec` So I thought I'd search for a zenity fork and I found there is [zenity-light](https://aur.archlinux.org/packages/zenity-light) that doesn't use webkit, works just fine! Perhaps this could be bundled with lutris own winetricks?

Thanks, zenity-light fixes lutris winetricks problem and doesn't break steam-manjaro (as it depends on zenity)

--- a/lutris/runners/commands/wine.py
+++ b/lutris/runners/commands/wine.py
@@ -342,8 +342,21 @@ def winetricks(
     winetricks_path = os.path.join(settings.RUNTIME_DIR, "winetricks/winetricks")
     if system_winetricks or not system.path_exists(winetricks_path):
         winetricks_path = system.find_executable("winetricks")
+        working_dir = None
         if not winetricks_path:
             raise RuntimeError("No installation of winetricks found")
+    else:
+        # We will use our own zentiy if available, which is here and it
+        # also needs a data file in this directory. We have to set the
+        # working_dir so it will find the data file.
+        working_dir = os.path.join(settings.RUNTIME_DIR, "winetricks")
+
+        if not env:
+            env = {}
+
+        path = env.get("PATH", os.environ["PATH"])
+        env["PATH"] = "%s:%s" % (working_dir, path)
+
     if wine_path:
         winetricks_wine = wine_path
     else:
@@ -355,11 +368,13 @@ def winetricks(
     args = app
     if str(silent).lower() in ("yes", "on", "true"):
         args = "--unattended " + args
+
     return wineexec(
         None,
         prefix=prefix,
         winetricks_wine=winetricks_wine,
         wine_path=winetricks_path,
+        working_dir=working_dir,
         arch=arch,
         args=args,
         config=config,

